frontends/concrete-python/concrete/fhe/__init__.py

```diff
@@ -193,3 +193,26 @@
 from .version import __version__
 
 # pylint: enable=import-error,no-name-in-module
+
+import warnings
+from cpuinfo import get_cpu_info
+
+def _is_cpu_compatible() -> bool:
+    """Check if CPU can run Concrete
+
+    Returns:
+        bool: if the CPU does support required features to run Concrete
+    """
+    flags = get_cpu_info()["flags"]
+    if "aes" in flags:
+        return True
+    else:
+        return False
+
+# Print a warning if the CPU can't run Concrete
+if not _is_cpu_compatible():
```

Comment:
why at top level ?
can't they at least compile but not run ?
if the case, maybe the warning should be an error when calling a circuit execution.
and onluy a warning when calling compilation

Reply:
The idea is to show a warning whenever you import concrete. It take some time to run so I'm not sure about adding it as part of the execution call